CodeEdit/Features/AppPreferences/Model/General/GeneralPreferences.swift

```diff
@@ -47,6 +47,9 @@ extension AppPreferences {
         /// The Issue Navigator Detail line limit
         var issueNavigatorDetail: NavigatorDetail = .upTo3
 
+        /// The file icon's opacity
+        var opacity: Double = 1.0
```

Comment:
Why do we need an opacity setting for the file icons?

Reply:
People might want to have white backgrounds by default.

Reply:
Sure, this is a niche case but as highlighted by [codeedit.app](codeedit.app) we want to have maximum customisation.

Reply:
What do you suggest otherwise?

Reply:
What do you mean with _white backgrounds_?

Reply:
By default `SF Symbols`'s backgrounds are **white** _or_ the system theme, that's what I meant 😄

Reply:
> SF Symbols don't have a background at all. They just have a foreground color. In special cases they can be `multi-color` or even `hierarchical`. Also the default color is `Color.primary` or `Color.accentColor` for regular components and interactive components respectively.

Maybe I have an older version of `SF Symbols`

Reply:
2, that's probably why.

Reply:
Or it could be the background colour